themes/default/content/learn/pulumi-fundamentals/create-a-pulumi-project/index.md

```diff
@@ -172,6 +198,14 @@ The basic project created by `pulumi new` is comprised of multiple files:
 
 {{< /choosable >}}
 
+{{< choosable language go >}}
+
+<li><code>go.mod</code>: a file that describes the Go program's properties</li>
+<li><code>go.sum</code>: a generated file that maintains the checksums for the packages imported by the Go program</li>
+<li>{{< langfile >}}: your program's main entrypoint file</li>
```

Comment:
@cnunciato It doesn't look like switching to `{{% %}}` delimiters and using Markdown is going to work; the `langfile` shortcode is adding a line return for some reason. I'll keep playing with it to see if I can make it work, but this may be why we were using HTML previously.

Reply:
I'm going to have to revert to HTML here; I can't find a way to make the `{{< langfile >}}` shortcode work without inserting an extra line. I'll file an issue for us to revisit this later.

Reply:
Does this work? https://github.com/pulumi/pulumi-hugo/blob/master/themes/default/content/docs/get-started/aws/review-project.md?plain=1#L46

Reply:
@cnunciato It might, but then we're still using HTML. 😄